src/hooks/use

```diff
@@ -0,0 +1,8 @@
+export default function useEvents() {
+  const handleChange = (
+    e: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLSelectElement>
+  ) => {};
+  return {
+    handleChange,
+  };
+}
```

Comment:
What is this?

Reply:
Here will be the functions handleChange and handleSubmit